docs/reference/search/search-template.asciidoc

```diff
@@ -343,7 +343,7 @@ We could write the query as:
 ==================================
 As written above, this template is not valid JSON because it includes the
 _section_ markers like `{{#line_no}}`.  For this reason, the template should
-either be stored in a file (see <<pre-registered-templates>>) or, when used
+either be stored in elasticsearch (see <<pre-registered-templates>>) or, when used
```

Comment:
```suggestion
either be stored in {es} (see <<pre-registered-templates>>) or, when used
```

`{es}` will expand to `Elasticsearch.` If you prefer not to use an attribute, please capitalize `Elasticsearch`.